app/src/views/OperationalLearning/index.tsx

```diff
@@ -98,6 +103,114 @@ const perComponentKeySelector = (option: PerComponent) => option.id;
 const disasterTypeKeySelector = (type: DisasterType) => type.id;
 const disasterTypeLabelSelector = (type: DisasterType) => type.name ?? '?';
 
+const responseData = {
+    operations_included: 9,
+    learning_extracts: 6,
+    sectors_covered: 6,
+    sources_used: 8,
+    learning_by_region: [
+        {
+            region_name: 'Americas',
+            region_id: 1,
+            count: 2,
+        },
+        {
+            region_name: 'Asia Pacific',
+            region_id: 2,
+            count: 5,
+        },
+        {
+            region_name: 'Europe',
+            region_id: 3,
+            count: 2,
+        },
+    ],
+    learning_by_sector: [
+        {
+            id: 17,
+            count: 1,
+            title: 'health',
+        },
+        {
+            id: 18,
+            count: 1,
+            title: 'education',
+        },
+        {
+            id: 19,
+            count: 3,
+            title: 'Livelihoods and basic needs',
+        },
+        {
+            id: 20,
+            count: 4,
+            title: 'Migration',
+        },
+        {
+            id: 21,
+            count: 1,
+            title: 'WASH',
+        },
+        {
+            id: 22,
+            count: 1,
+            title: 'Shelter',
+        },
+    ],
+    sources_overtime: {
+        DREF: [
+            { year: 2023, count: 1 },
+            { year: 2024, count: 3 },
+        ],
+        'Emergency Appeal': [
+            { year: 2023, count: 1 },
+            { year: 2024, count: 1 },
+        ],
+        'International Appeal': [
+            { year: 2023, count: 1 },
+            { year: 2024, count: 1 },
+        ],
+        'Forecast Based Action': [
+            { year: 2022, count: 1 },
+        ],
+    },
+};
+
+const mockSectorData = responseData.learning_by_sector.map((sector) => ({
+    key: sector.id,
+    value: sector.count,
+    label: sector.title,
+}));
+
+const mockRegionData = responseData.learning_by_region.map((region) => ({
+    key: region.region_id,
+    value: region.count,
+    label: region.region_name,
+}));
```

Comment:
We don't need to transform this. Just change the key, value and label selector to match the initial data.